### DISCUSSION TITLE
How to distritbute to internal and external group with pilot

### DISCUSSION THREAD

**Discussion by engelkes-finstreet** [fastlane/fastlane#20796] [Q&A]
Hey,
I really cannot figure out how to distribute my application to an internal **and** external group with pilot. 
For reference I have one internal group with the name `company-intern` and one group with the name `company-extern` which are internal and external groups respectively. 

My understanding of the docs is that I can add an array to the `groups` parameter and add all the groups I want to distribute the app to. Here my code with the pilot passage:

```
pilot(
  apple_id: "#{DEVELOPER_APP_ID}",
  app_identifier: "#{DEVELOPER_APP_IDENTIFIER}",
  distribute_external: true,
  groups: ["company-intern", "company-extern"],
  skip_submission: true,
  ipa: "./Runner.ipa",
  changelog: ""
)    
``` 

With this the app is only distributed to the `company-intern` group. Can somebody help me how to distribute the app to both groups?

**Comment by enlivn** [comment]
@engelkes-finstreet is your `changelog` actually empty or is your snippet just an example? See this code: 

https://github.com/fastlane/fastlane/blob/ec78d7d81a6a66e9e89fd29f6e0616d5ba09840a/pilot/lib/pilot/build_manager.rb#L102-L110
